src/GraphQl/Subscription/SubscriptionIdentifierGenerator.php

```diff
@@ -23,7 +23,21 @@ final class SubscriptionIdentifierGenerator implements SubscriptionIdentifierGen
     public function generateSubscriptionIdentifier(array $fields): string
     {
         unset($fields['mercureUrl'], $fields['clientSubscriptionId']);
+        $fields = $this->removeTypename($fields);
```

Comment:
why is this needed?

Reply:
When executing a subscription GraphQL query, the __typename key is not present in fields in the SubscriptionProcessor, but when you get fields in doctrine subscriber to publish updates, the __typename is present in the fields. That results in different sha256 keys despite you wanting the same thing.